jsk_spot_robot/jsk_spot_driver.rosinstall.noetic

```diff
@@ -0,0 +1,68 @@
+# spot-ros is required for spot
+# This is a develop branch for jsk version.
+# We need to use it until it is merged to master
+# - git:
+#     local-name: spot-ros
+#     uri: https://github.com/sktometometo/spot_ros.git
+#     version: develop/spot
+## use https://github.com/clearpathrobotics/spot_ros/pull/67 (Commands for the Arm and gripper, and URDF and simulation of the arm added)
+# - git:
+#     local-name: spot-ros
+#     uri: https://github.com/estherRay/spot_ros.git
+#     version: b5fe5e5c4a732c1a724e4f86abc2be6395088d02
+## use https://github.com/cst0/spot_ros/pull/4 for Arm and gripper
+- git:
+    local-name: spot-ros
+    uri: https://github.com/k-okada/spot_ros-arm.git
+    version: 0a8c4b29768ae946d7c84aec9a2ae2d185ed97a9
+## to support custom limb name, we need https://github.com/jsk-ros-pkg/jsk_model_tools/pull/249
+- git:
+    local-name: jsk_model_tools
+    uri: https://github.com/k-okada/jsk_model_tools.git
+    version: custom_limb
+#
+# use robot_upstart to install supervisor (https://github.com/clearpathrobotics/robot_upstart/pull/113)
+- git:
+    local-name: robot_upstart
+    uri: https://github.com/k-okada/robot_upstart.git
+    version: develop
+# aques_talk needs to be compiled from source
+- tar:
+    local-name: aques_talk
+    uri: https://github.com/tork-a/jsk_3rdparty-release/archive/refs/tags/release/melodic/aques_talk/2.1.24-2.tar.gz
+# use parallel app_manager
+- git:
+    local-name: app_manager
+    uri: https://github.com/PR2/app_manager.git
+    version: kinetic-devel
+# wait for https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/443 (add volume key arg for robot-interface :speak) to be released
+- git:
+    local-name: jsk_pr2eus
+    uri:     https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+    version: master
+#
+#
+# need to add following code to  ublox_gps/config/zed_f9p.yaml
+#config_on_startup: true
+#
+#rate: 0.016
+## TMODE3 Config
+#tmode3: 1                   # Survey-In Mode
+#sv_in:
+#  reset: false              # True: disables and re-enables survey-in (resets)
+#                            # False: Disables survey-in only if TMODE3 is
+#                            # disabled
+#  min_dur: 300              # Survey-In Minimum Duration [s]
+#  acc_lim: 3.0              # Survey-In Accuracy Limit [m]
+- git:
+    local-name: ublox
+    uri: https://github.com/KumarRobotics/ublox.git
+# insta360 need latest jsk_perception
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri:    jsk-ros-pkg/jsk_recognition.git
+    version: master
+# Panoram image recognition need coral_usb, which is not release as apt package.
```

Comment:
coral_usb does not have apt for melodic too. So how do we setup the environment on melodic https://github.com/k-okada/jsk_robot/blob/spot_arm/jsk_spot_robot/SetupInternalPCAndSpotUser.md#prerequities ?
I prefer to maintain one document for both 18.04/20.04, to keep system with less resources.